doppler/assets/oauth_clients.json

```diff
@@ -0,0 +1,13 @@
+{
+  "integration": {
+    "name": "Doppler API",
+    "client_role": "integration",
+    "onboarding_url": "https://dashboard.doppler.com/notification_services/datadog/connect",
+    "redirect_uris": [
+      "https://dashboard.doppler.com/notification_services/datadog/callback"
+    ],
+    "description": "Doppler API Client",
+    "id": "87fdb7f6-542c-11ef-abb5-da7ad0900002",
+    "scopes": ["api_keys_write", "agent_upgrade_write"]
```

Comment:
please remove the agent_upgrade write scope from your oauth client and re-publish the client

Reply:
Done!